bin/tanner

```diff
@@ -17,7 +17,9 @@ def main():
     """)
     parser = argparse.ArgumentParser()
     parser.add_argument("--config", help="tanner config")
+    parser.add_argument("--database", help="choose a database postgres/redis", default='redis')
```

Comment:
Since we agree with @glaslos that we should use only postgresql for now, it's not necessary to set db as a parameter

Reply:
Ok :+1: ill remove that.